### DISCUSSION TITLE
How to upgrade "nextra": "^2.0.0-beta.43" to nextra@2.0.0 stable ?

### DISCUSSION THREAD

**Discussion by Yuzuki-Matsumoto** [shuding/nextra#1003] [General]
### For those who have already used the beta version, please mention how to upgrade

Got multiple errors while trying to upgrade "next": "^12.2.3", to "next": "^13.0.5", (Many components are not working like features)
My `package.json`
```
{
  "name": "docs",
  "scripts": {
    "preinstall": "npx only-allow pnpm",
    "dev": "next",
    "start": "next start",
    "build": "next build"
  },
  "author": "Ytiam",
  "license": "MIT",
  "dependencies": {
    "@sentry/nextjs": "^7.17.3",
    "@vercel/og": "^0.0.20",
    "axios": "0.27.2",
    "classnames": "2.3.2",
    "clsx": "^1.2.1",
    "framer-motion": "^7.3.2",
    "next": "^12.2.3",
    "nextra": "^2.0.0-beta.43",
    "nextra-theme-docs": "2.0.0-beta.25",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@types/node": "^18.7.8",
    "autoprefixer": "^10.4.8",
    "postcss": "^8.4.16",
    "tailwindcss": "^3.1.8",
    "typescript": "^4.7.4"
  }
}

**Comment by shuding** [comment]
Hi! I think there are these breakings:

- some renamed config names (https://github.com/shuding/nextra/pull/721), you can take a look at the latest types to know more
-  `meta.json` is now `_meta.json`
- components will need to be imported like `import { Callout } from 'nextra-theme-docs'`

**Reply by mi-hol** [reply]
- another breaking change is https://github.com/shuding/nextra/pull/595
old syntax to get full year of current date does no longer work. 
What is the replacement for `{new Date().getFullYear()}`?
update: found syntax in https://nextra.site/docs/docs-theme/theme-configuration#copyright-information

> * components will need to be imported like `import { Callout } from 'nextra-theme-docs'`

Could you please elaborate on the detailed change an existing project needs to make to implement these changes correctly (where and what) ?

**Reply by mi-hol** [reply]
> > * components will need to be imported like `import { Callout } from 'nextra-theme-docs'`
> 
> Could you please elaborate on the detailed change an existing project needs to make to implement these changes correctly (where and what) ?

Could someone please comment? 
The [documentation I found](https://nextra.site/docs/docs-theme/built-ins/callout) leaves the details open at least for a non-next.js programmer :(

I've added import to my previously working .md file but that gives below error

![image](https://user-images.githubusercontent.com/22799428/206907344-530b9ecb-9bf6-4d15-a74a-505ba48c8ccc.png)
![image](https://user-images.githubusercontent.com/22799428/206907494-16b2a105-7d67-4153-8aa9-4964ceed2ee4.png)

**Reply by mi-hol** [reply]
solution was to replace "import Callout from 'nextra-theme-docs/callout'" with "import { Callout } from 'nextra-theme-docs'"
